File: src/main/java/com/example/appforauslenderamt/entity/HealthInsuranceInfo.java
package com.example.appforauslenderamt.entity;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.Value;

@Value
@Builder
@Getter
@AllArgsConstructor
public class HealthInsuranceInfo {
    @JsonProperty("has_health_insurance")
    Boolean hasHealthInsurance;
    @JsonProperty("insurer")
    String insurer;
}

File: Frontend/src/app/Models/BackendModels/HealthInsuranceInfo.ts
import { Expose } from "class-transformer"

export class HealthInsuranceInfo {
   @Expose()
   has_health_insurance: boolean
   @Expose()
   insurer:string
     constructor(hasInsurance: boolean, insurer: string){
        this.insurer = insurer
        this.has_health_insurance = hasInsurance
     }
}
